packages/x-charts/src/ChartsVoronoiHandler/ChartsVoronoiHandler.tsx

```diff
@@ -180,11 +192,11 @@ function ChartsVoronoiHandler(props: ChartsVoronoiHandlerProps) {
       onItemClick(event, { type: 'scatter', seriesId, dataIndex });
     };
 
-    element.addEventListener('pointerout', handleMouseOut);
+    element.addEventListener('pointerleave', handleMouseLeave);
```

Comment:
`out`>`leave` as `out` triggers on every element inside the chart, rather than leaving the main svg